src/tests/test_scraper_base.py

```diff
@@ -21,6 +22,14 @@ def test_postgres_from_env(self):
             assert test_db.get_db_url().startswith("postgres")
 
 
+class TestGeocoder(TestCase):
```

Comment:
What does a geocoder test has to do in a scrapers test file? ;) It's not even in the same module.